### ISSUE TITLE
Avoid underlined text that could look like a link

### ISSUE THREAD

**Issue by alvaromontoro** [jackdomleo7/Checka11y.css#51]
### Describe the new a11y feature or project enhancement

The `<u>` tag used to represent an underlined chunk of text. Since HTML5, its semantic meaning has changed (now it represents text that has some non-textual annotation associated), but many times it is still used just to underline text because its default styles include an underline. 

This is an accessibility problem because, in general, an underlined piece of text is used to represent a link. So the `<u>` tag should be avoided. From MDN: "_Be careful to avoid using the `<u>` element with its default styling (of underlined text) in such a way as to be confused with a hyperlink, which is also underlined by default._"

### Describe the solution you'd like

If there is a `<u>` tag on the page, show a warning as it is most likely an incorrect use of the tag. The CSS selector could be as simple as this:

```css
u {
  /* warning */
}
```

In the warning, a list of alternatives to `<u>` could be provided. From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u#Other_elements_to_consider_using):

> In most cases, you should use an element other than `<u>`, such as:
> - `<em>` to denote stress emphasis
> - `<b>` to draw attention to text
> - `<mark>` to mark key words or phrases
> - `<strong>` to indicate that text has strong importance
> - `<cite>` to mark the titles of books or other publications
> - `<i>` to denote technical terms, transliterations, thoughts, or names of vessels in Western texts

### Link(s)

- [MDN: the `<u>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u#Other_elements_to_consider_using)

**Comment by shonamckenzie** [comment]
Hi @jackdomleo7 @alvaromontoro if you think this may be a good first issue then I'm happy to take a look and see if I can get a PR together
